server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,229 @@
+   cbas_request_body = {
+      "appType": "CROMWELL"
+   } 
+        
+   response = requests.post(url=activate_cbas_request, json=cbas_request_body, 
+                            headers={"Authorization": f"Bearer {bearer_token}"})
+   # will return 202 or error
+   handle_failed_request(response, "Error activating CBAS", 202)
+   output_message("CBAS successfully activated")
+   return create_workspace_data
+
+# GET WDS OR CROMWELL ENDPOINT URL FROM LEO
+def get_app_url(workspaceId, app):
+    uri = f"{leo_url}/api/apps/v2/{workspaceId}?includeDeleted=false"
+
+    headers = {"Authorization": f"Bearer {bearer_token}",
+               "accept": "application/json"}
+
+    response = requests.get(uri, headers=headers)
+    status_code = response.status_code
+    handle_failed_request(response, f"{status_code} - Error retrieving details for {workspaceId}", 200)
+    output_message(f"Successfully retrieved details.\n{response.text}")
+    response = response.json()
+
+    app_url = ""
+    app_type = "CROMWELL" if app != 'wds' else app.upper()
+    output_message(f"App type: {app_type}")
+    for entries in response: 
+        if entries['appType'] == app_type and entries['proxyUrls'][app] is not None:
+            if(entries['status'] == "PROVISIONING"):
+                output_message(f"{app} is still provisioning")
+                break
+            output_message(f"App status: {entries['status']}")
+            app_url = entries['proxyUrls'][app]
+            break 
+
+    if app_url is None: 
+        output_message(f"{app} is missing in current workspace")
+    else:
+        output_message(f"{app} url: {app_url}")
+
+    return app_url
+
+def submit_workflow_to_cromwell(app_url, workflow_test_name):
+    absolute_file_path = os.path.dirname(__file__)
+    workflow_source_path = os.path.join(absolute_file_path, './workflow_files/hello.wdl')
+    workflow_inputs_path = os.path.join(absolute_file_path, './workflow_files/hello.inputs')
+    workflow_endpoint = f'{app_url}/api/workflows/v1'
+    headers = {"Authorization": f'Bearer {bearer_token}',
+              "accept": "application/json",
+    }
+    with open(workflow_source_path, 'rb') as hello_wdl:
+        with open(workflow_inputs_path, 'rb') as hello_inputs:
+            files = {
+                'workflowSource': ('hello.wdl', hello_wdl, 'application/octet-stream'),
+                'workflowInputs': ('hello.inputs', hello_inputs, 'application/octet-stream'),
+                'workflowType': 'WDL',
+                'workflowTypeVersion': '1.0'
+            }
+            response = requests.post(workflow_endpoint, headers=headers, files=files)
+            handle_failed_request(response, f"Error submitting workflow to Cromwell for {workflow_test_name}", 201)
+            output_message(response.json())
+            return response.json()
+
+def get_workflow_information(app_url, workflow_id):
+    workflow_endpoint = f'{app_url}/api/workflows/v1/{workflow_id}/status'
```

Comment:
Not a blocker, but might be nice to pull all these REST calls out into a `make_request` method. Maybe better to leave for now and use or create a helper method in the shared repo.